net/transmission/files/transmission.init

```diff
@@ -164,6 +168,8 @@ transmission() {
 	procd_add_jail_mount_rw "$config_dir/blocklists"
 	procd_add_jail_mount_rw "$config_dir/stats.json"
 	procd_add_jail_mount_rw "$download_dir"
+	procd_add_jail_mount_rw "$incomplete_dir"
+	procd_add_jail_mount_rw "$watch_dir"
```

Comment:
Similar here, should be conditionally added only if `[ "$watch_dir_enabled" = "1" ]`.

Reply:
fixed